spec/lib/defra_validators/validators/defra_validators/companies_house_name_validator_spec.rb

```diff
@@ -0,0 +1,123 @@
+# rubocop:disable Style/FrozenStringLiteralComment
+require "spec_helper"
+
+module DefraValidators
+  RSpec.describe CompaniesHouseNameValidator do
+    class NameValidatable
+      include ActiveModel::Model
+      include ActiveModel::Validations
+
+      attr_accessor :name
+
+      validates :name, "defra_validators/companies_house_name": true
+    end
+
+    let(:disallowed_characters) do
+      [
+        "^",
+        "|",
+        "_",
+        "~",
+        "¬",
+        "`"
+      ]
+    end
+
+    describe ".disallowed_chars" do
+      subject { described_class }
+
+      it "should contain the expected disallowed characters" do
+        expect(subject.disallowed_chars)
+          .to eql(disallowed_characters)
+      end
+    end
+
+    context "validations" do
+      subject { NameValidatable.new(name: name) }
+
+      describe "#valid?" do
+        context "field with alphanumeric characters" do
+          let(:name) { "Joe Bloggs 123456 Ltd" }
+
+          it { is_expected.to be_valid }
+        end
+
+        context "field contains only numbers" do
+          let(:name) { "123456" }
+
+          it "is valid" do
```

Comment:
Any reason for the change in format from the previous test, as in not using

```suggestion
          it { is_expected.to be_valid }
```

If there is a reason, awesome. Else I'd push to make both consistent

Reply:
I think this is one from copying and pasting the original tests 🤔 

I agree, they should be consistent.